gslib/tests/util.py

```diff
@@ -623,14 +623,19 @@ def MakeBucketNameValid(name):
   """Returns a copy of the given name with any invalid characters replaced.
 
   Args:
-    name (str): The bucket name to transform into a valid name.
+    name (str, unicode, bytes): The bucket name to transform into a valid name.
 
   Returns:
-    (str) The version of the bucket name containing only valid characters.
+    (str, unicode, bytes) The version of the bucket name containing only
+      valid characters.
   """
   # Neither underscores nor uppercase letters are valid characters for a
   # bucket name. Replace those with hyphens and lowercase characters.
-  return name.replace('_', '-').lower()
+  if isinstance(name, (six.text_type, six.binary_type)):
+    return name.replace('_', '-').lower()
+  else:
+    raise TypeError("Unable to format name. Incorrect Type: {0}".format(
```

Comment:
Nit: single quotes instead of double quotes here.

Reply:
Changed